tests/tests.entry.ts

```diff
@@ -0,0 +1,23 @@
+import "core-js";
+import "core-js/es7/reflect";
+import "zone.js/dist/zone";
+import "zone.js/dist/long-stack-trace-zone";
+import "zone.js/dist/async-test";
+import "zone.js/dist/fake-async-test";
+import "zone.js/dist/sync-test";
+import "zone.js/dist/proxy";
+import "zone.js/dist/jasmine-patch";
+
+const browserTesting = require("@angular/platform-browser-dynamic/testing");
+const coreTesting = require("@angular/core/testing");
+// include every .spec.ts file inside src, except for those in ks-app folder
+const context = (require as any).context("../src/", true, /^((?![\\/]ks-app[\\/]).)*\.spec\.ts$/);
```

Comment:
Ugh. This makes me wonder if having the ks-app in here is going to be more trouble than its worth.

Reply:
@jeeyun @hippee-lee: I don't clearly understand the benefits of having the ks-app and the website in the master branch.

I have always liked things not related to our main code base separate (website for eg). This way reviews are not needed for website for small changes and the person publishing the website is not blocked. Same for the ks-app.

Reply:
@adityarb88 - I see your point. Imo - they don't belong in a separate branch either though.

Reply:
@hippee-lee: Trying to understand why though. Both the Kitchen Sink app and the website consume the build and have nothing to do with the source code right? So if thats the case then I am not exactly sure why we need to have them in the `master` branch.

Reply:
Sorry my comment wasn't clear enough. When I meant is that I do agree with you, but, perhaps since they are separate projects/codebases they should have their own repo. 

Here are a couple of thoughts about how we might move forward with decoupling them

- Keep the demo app (or ks-app with proper sourcemapping back to Clarity added) - we will need something to develop / debug components against 
- Move the app for regression testing into its own repo. Regression tests should still live in Clarity but the app it tests doesn't have to. Its feasible that we could publish it to a server (or surge or etc ...) and change the tests to point the browser in selenium-standalone there
- Move the website docs into it's own repo so that it has its own history, tags

> I have always liked things not related to our main code base separate 
Me too, but I don't think they should be on a separate branch in the same repo, I think they should have their own repo.

Reply:
my two cents:
- I prefer having the `ks-app` in master. This is nice as we don't have to check out another project or switch to any branch to test out our npm packages
- I agree with your points on that it adds additional headache for `ks-app` to live under the `src` folder though.. is it possible for the `ks-app` to live in master but outside `src` folder? Or does this violate (philosophically) the structure we want?

Reply:
I will investigate that after we make the switch to webpack 😀